src/bin/psi4/read_options.cc

```diff
@@ -1210,6 +1210,8 @@ int read_options(const std::string &name, Options & options, bool suppress_print
     (if set), or else by the name of the output file plus the name of
     the current molecule. -*/
     options.add_bool("MOLDEN_WRITE", false);
+    /* Write all the MOs to the MOLDEN file (false) or discard the unoccupied MOs (true). */
+    options.add_bool("MOLDEN_NOVIRTUAL", false);
```

Comment:
For booleans, I've found it easier to reason with if they convey positive instructions. Maybe switch the logic to `MOLDEN_WITH_VIRTUAL` and default `true`.